tests/integration/API/GetWorks.test.ts

```diff
@@ -0,0 +1,158 @@
+/* tslint:disable:no-relative-imports */
+import { Claim, isClaim } from '@po.et/poet-js'
+import { all, pickBy } from 'ramda'
+import { describe } from 'riteway'
+
+import {
+  ABraveAndStartlingTruth,
+  AllGodsChildrenNeedTravelingShoes,
+  ASongFlungUpToHeaven,
+  AStudyInScarlet,
+  GatherTogetherInMyName,
+  KnowWhyTheCagedBirdSings,
+  MomAndMeAndMom,
+  OnThePulseOfMorning,
+  SinginAndSwinginAndGettingMerryLikeChristmas,
+  TheHeartOfAWoman,
+  TheMurdersInTheRueMorgue,
+  TheRaven,
+  TheWeekOfDiana,
+} from '../../helpers/Claims'
+import { delay, runtimeId, setUpServerAndDb } from '../../helpers/utils'
+import { getWork, postWorkWithDelay } from '../../helpers/works'
+
+const PREFIX = `test-functional-node-poet-${runtimeId()}`
+const NODE_PORT = '28081'
+const getWorkFromNode = getWork(NODE_PORT)
+const postWorkToNode = postWorkWithDelay(NODE_PORT)
+
+const setUpExistingClaims = async (claims: ReadonlyArray<Claim>) =>
+  await Promise.all(claims.map(async (claim: Claim) => postWorkToNode(claim)))
+
+const works = [
+  ABraveAndStartlingTruth,
+  AllGodsChildrenNeedTravelingShoes,
+  ASongFlungUpToHeaven,
+  AStudyInScarlet,
+  GatherTogetherInMyName,
+  KnowWhyTheCagedBirdSings,
+  MomAndMeAndMom,
+  OnThePulseOfMorning,
+  SinginAndSwinginAndGettingMerryLikeChristmas,
+  TheHeartOfAWoman,
+  TheMurdersInTheRueMorgue,
+  TheRaven,
+  TheWeekOfDiana,
+]
+
+const getClaimWithoutTimestamp = (claim: any) => pickBy((v: any, k: string) => k !== 'timestamp', claim)
+const getClaimsWithoutTimestamps = (claims: ReadonlyArray<any>) => claims.map(getClaimWithoutTimestamp)
+
+describe('GET /works', async assert => {
+  // Setup Mongodb and the app server
+  const { db, server } = await setUpServerAndDb({ PREFIX, NODE_PORT })
+
+  // Wait for everything to finish starting
+  await delay(5 * 1000)
+
+  // Setup Existing Claims
+  await setUpExistingClaims(works)
+
+  {
+    const response = await getWorkFromNode('')
+    const claims = await response.json()
+
+    assert({
+      given: 'a call to GET /works',
```

Comment:
Have you considered doing

```ts
const given = 'a call to GET /works'

assert({
  given,
  should: 'return a status of 200',
  actual: response.status,
  expected: 200,
})
```

To avoid copy-paste?

Reply:
🥇 Obviously not.... 😀